CHANGES_UNRELEASED.md

```diff
@@ -25,6 +25,7 @@ Use the template below to make assigning a version number during the release cut
 ## Places
 ### What's new
   - We now expose all of the Places history APIs to Swift consumers. ([#4989](https://github.com/mozilla/application-services/pull/4989))
+  - Added an optional db_size_limit parameter to `run_maintanance`.  This can be used to set a target size for the places DB.  If the DB is over that size, we'll prune a few older visits. The number of visits is very small (6) to ensure that the operation only blocks the database for a short time. The intention is that `run_maintanaince()` is called frequently compared to how often visits are added to the places DB.
```

Comment:
lol - an even more creative spelling of "maintenance" here now :) It's spelt incorrectly twice here now :) Ditto below, so maybe a search-replace for "maintanance"?